src/index.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+
+import fs from 'fs-extra'
+import moment from 'moment'
+
+/**
+ * Utility for caching to disk
+ * @class
+ */
+export default class SimpleCache {
+  /**
+   * gets or makes values for a cache
+   * @param {string} cachePath - cache path
+   * @param {number} cacheDuration - cache validity in seconds
+   * @param {any} cacheFn - function, returns json to cache
+   * @example
+   * ```js
+   * const cache = require('easy-money')
+   * await cache.fetch('/path/to/cache', 3600, cacheFn)
+   * ```
+   */
+  static async fetch (cachePath: string, cacheDuration: ?number, cacheFn: any): Promise<?Array<?string>> {
```

Comment:
The types should be fixed here. I'm not sure if this works, but it should be something like this:

```
static async fetch <T> (cachePath: string, cacheDuration: ?number, cacheFn: () => T): Promise<T> {
}
```